src/overlays/actors/ovl_En_Sth/z_en_sth.c

```diff
@@ -630,59 +672,65 @@ void func_80B680A8(Actor* thisx, PlayState* play) {
 
     this->actionFunc(this, play);
 
-    if (func_80B6703C(this, play) && !(this->unk_29C & 8) && (this->unk_29A != 5)) {
-        sp38.x = sp38.y = sp38.z = 0;
+    if (EnSth_DetectSpeakingRange(this, play) && !(this->sthFlags & STH_FLAG_DISABLE_HEAD_TRACK) &&
+        (this->curAnimIndex != STH_ANIM_LOOKING_AROUND)) {
+        Vec3s torsoRot;
+        torsoRot.x = torsoRot.y = torsoRot.z = 0; // this should block torso rot from working
```

Comment:
Blank line between variable declaration and code:
```suggestion
        Vec3s torsoRot;
        
        torsoRot.x = torsoRot.y = torsoRot.z = 0; // this should block torso rot from working
```